saltpylint/thirdparty.py

```diff
@@ -83,46 +80,37 @@ def __init__(self, linter=None):
         self._inside_try_except = False
         self._inside_funcdef = False
         self._inside_if = False
-        self.cwd = self.allowed_3rd_party_modules = None
+        self.cwd = self.allowed_3rd_party_modules = []
 
     def open(self):
         super(ThirdPartyImportsChecker, self).open()
         self.cwd = os.getcwd()
-        self.allowed_3rd_party_modules = set(self.config.allowed_3rd_party_modules)  # pylint: disable=no-member
```

Comment:
Why don't we want a set?
The set would remove duplicates

Reply:
i add it back in